Files/Local Files/Copy a Single file from Fileshare to SFTP/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Copy a Single file from Fileshare to SFTP",
+  "version": "1.0.0",
+  "tags": [
+    "Files",
+    "SFTP",
+    "Fileshare"
+  ],
+  "additionalSystemsUsed": [
+    "SFTP"
+  ],
+  "tasks": [
+     "Frends.Files.Read.Files.Read",

```

Comment:
No need to have the tasks part in here - this will get populated automatically in template portal based on the process.json file. So you can remove the thole tasks property.

Reply:
Changes made as specified